WordPress/src/main/java/org/wordpress/android/ui/prefs/categories/CategoriesListFragment.kt

```diff
@@ -0,0 +1,33 @@
+package org.wordpress.android.ui.prefs.categories
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.SiteSettingsCategoriesListFragmentBinding
+import javax.inject.Inject
+
+class CategoriesListFragment : Fragment(R.layout.site_settings_categories_list_fragment) {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var viewModel: CategoriesViewModel
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(SiteSettingsCategoriesListFragmentBinding.bind(view)) {
+            initDagger()
+            initViewModel()
+        }
+    }
+
+    private fun initDagger() {
+        (requireActivity().application as WordPress).component()?.inject(this)
+    }
+
+    private fun SiteSettingsCategoriesListFragmentBinding.initViewModel() {
```

Comment:
Minor (🔍): The `SiteSettingsCategoriesListFragmentBinding` received parameter is never used. I understand that you might did that on purpose because a subsequent PR will start using that anyway, but for the time being it still shows as a warning, which is still unknown as for why, at least to the reviewer. Or, maybe you might actually not need it at all, I am not sure.

In the first case, if it was me, if I was expecting that I will be needing it, then what I would have done is to be explicit about it and suppress it to convey my indentions, with `@Suppress("unused")` close to the `initViewModel()` function. Then, when I end up writing the code that needs that, I would first the suppression. Wdyt?

PS: As you can tell I don't like warning that much... 😝

Reply:
Thanks for the input, but this is the skeleton - next PR uses it and is coming today, so it's staying as is. 😉  

> which is still unknown as for why, at least to the reviewer

I can see that, but the PR description states it's just the skeleton, which I know you are familiar with. It's placeholder code.